### DISCUSSION TITLE
WINASIO

### DISCUSSION THREAD

**Discussion by williamcorney** [The-Wineskin-Project/WineskinServer#326] [Q&A]
Hello

I am using your beautiful product and just have this one nagging issue.

i want to configure winasio but dont see how to correctly create/inject/import the winasio.dll into my wineskin application.

I cant seem to 'build' the file myself as get an error about objbase.h file not found

I cant seem to download the file precompiled . The closest ive come is finding winasio.dll.so but im a bit clueless what exactly im supposed to do with the file. To rename it or not, whether to put in c_drive/windows/system32 or to put it contents/framework or sharedsupport/wine/. I've tried putting in locations such as these and also refreshing the wrapper but when i run the regsvr32 command to register winasio.dll get an error about failed to load

**Comment by Gcenx** [comment]
Apologies for not responding sooner.

I’m not sure if that would even work but I do see a JACK audio driver for macOS 10.12 and later so you’d firstly need that installed.
